Forge.TreeWalker/src/ITreeSession.cs

```diff
@@ -41,5 +41,11 @@ public interface ITreeSession
         /// </summary>
         /// <returns>The ActionResponse data for the last executed tree action key if it exists, otherwise null.</returns>
         Task<ActionResponse> GetLastActionResponseAsync();
+
+        /// <summary>
+        /// Get the string context if the actions in this TreeNodes are skipped.
+        /// Usually, when there is no action skipped, the return value is null.
```

Comment:
Gets the string context if the actions in the current TreeNode were skipped, or null if actions were not skipped.

Reply:
fixed

---
In reply to: [505844640](https://github.com/microsoft/Forge/pull/32#discussion_r505844640) [](ancestors = 505844640)